src/components/CorporateCampaign/CampaignProgressBar.vue

```diff
@@ -47,6 +47,32 @@
 			</kv-ui-button>
 		</div>
 	</div>
+	<div
+		v-else
```

Comment:
@dyersituations I was having trouble with the following code line `v-else-if="(basketLoans.length > 0) && isMatchingCampaign` so I changed it to `v-if=(promoAccount == promoAccount > 0) && !isMatchingCampaign` because I reread the ticket and I believe MatchingCampaigns are not supposed to have any promo credit at all ?